templates/main-page.html

```diff
@@ -1,24 +1,47 @@
-{% extends "auth.html" %}
-
-{% block body %}
+{% extends "auth.html" %} {% block body %}
 <div class="px-8 py-12 relative text-xl">
-    <a class="github-fork-ribbon left-top z-10" href="https://github.com/felienne/hedy" data-ribbon="Fork me on GitHub" title="Fork me on GitHub">Fork me on GitHub</a>
+    <a
+        class="github-fork-ribbon left-top z-10"
+        href="https://github.com/felienne/hedy"
+        data-ribbon="Fork me on GitHub"
+        title="Fork me on GitHub"
+        >Fork me on GitHub</a
+    >
     <div class="px-8">
-      <div class="-mx-16 -my-12 px-16 py-8 mb-8 bg-cover flex items-center" style="background-image: url(/images/header.jpg); height: 250px; position: relative;">
-        <div class="flex-1">
-            <h1 class="font-bold font-slab text-white text-6xl text-shadow-md tracking-wide">{{_('main_title')}}</h1>
-            <h2 class="font-sans font-light text-white text-shadow-md tracking-wide my-1">{{_('main_subtitle')}}</h2>
+        <div
+            class="-mx-16 -my-12 px-16 py-8 mb-8 bg-cover flex items-center"
+            style="
+                background-image: url(/images/header.jpg);
+                height: 250px;
+                position: relative;
+            "
+        >
+            <div class="flex-1">
+                <h1
+                    class="font-bold font-slab text-white text-6xl text-shadow-md tracking-wide"
+                >
+                    {{_('main_title')}}
+                </h1>
+                <h2
+                    class="font-sans font-light text-white text-shadow-md tracking-wide my-1"
+                >
+                    {{_('main_subtitle')}}
+                </h2>
+            </div>
+            <div class="flex-none">
+                <a href="/hedy"
+                    ><button class="green-btn text-white px-8 py-4">
+                        {{_('try_it')}}
+                    </button></a
+                >
+            </div>
```

Comment:
Same, this HTML code is very hard to read. Especially with the newlines with single `>` symbols.

Reply:
better?